pkg/conda/meta/testdata/README.md

```diff
@@ -0,0 +1,30 @@
+To recreate test files, run:
+```bash
+conda create --yes -n test-dep-parser python=3.8.8
```

Comment:
If i understand correctly -  there are 4 test cases:
1) normal library
2) library without license
3) library without name/version 
4) invalid json

I don't think we need a lot of test files.
We can install `libgomp=11.2.0` and get 2 files (for steps 1 and 2)

Reply:
Reduced the number of test files